analytics/lib/counts.py

```diff
@@ -113,7 +115,7 @@ def process_count_stat(stat: CountStat, fill_to_time: datetime) -> None:
         logger.info("START %s %s" % (stat.property, currently_filled))
         start = time.time()
         do_update_fill_state(fill_state, currently_filled, FillState.STARTED)
-        do_fill_count_stat_at_hour(stat, currently_filled)
+        do_fill_count_stat_at_hour(stat, currently_filled, realm)
```

Comment:
So, the code as of this commit is such that it deserves a comment that the code is in an intermediate state where we're only updating the count_stat for a single realm while updating the global FillState, that this will be fixed in future commits, and until then the `realm` argument is here just for plumbing, not for production use.

Reply:
I think that's what I want to get cleared as I mentioned in [comment](https://github.com/zulip/zulip/pull/13608#issuecomment-572023210). I wanted to know what should be the correct way handling global FillState should we add a realm argument there or there is a better approach than it. For now, I will comment it there.

Reply:
I think we're going to write future commits later in this effort that will adjust the model to support a per-realm FillState. 
 We haven't designed the model for that yet, so for now a comment reminding us that we'll need to do something there is the best we can do.

Reply:
(Or we might go with a solution that just involves skipping this step when we have a realm specified?  Again, I'm not sure, and want to defer designing that until we have this refactoring merged).